docker/docker-compose-dev.yml

```diff
@@ -3,27 +3,29 @@ services:
 
   actinia:
     build:
-      context: actinia-core-dev/
+      context: ..
```

Comment:
Just for my understanding: This means we look at the entire `actinia_core` directory?

Reply:
Yes, the whole repository is used as build context now. This is needed because the Dockerfile was adjusted to copy the whole source code inside and it doesn't work exclusively by mounting the code anymore.